extension/src/setup.ts

```diff
@@ -166,15 +169,19 @@ const warnUserCLIInaccessible = async (
     return
   }
   const response = await Toast.warnWithOptions(
-    'An error was thrown when trying to access the CLI.',
+    'An error was thrown when trying to access the CLI. Please ensure the correct interpreter is set for auto Python environment activation.',
```

Comment:
[I] Needs to know whether the Python extension is installed and mutate the options accordingly